### DISCUSSION TITLE
Welcome to Dishwashers Discussions!

### DISCUSSION THREAD

**Discussion by heiwot26** [heiwot26/Dishwashers#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by heiwot26** [comment]
Hey guys I have written up a doc on git hub flows and how to make pull requests. It's here if anyone needs it, I hope it's helpful!
https://1drv.ms/w/s!AoczMFihKuUfiVye0EE7f-MiwSka?e=stcDY7

**Reply by Nergez** [reply]
Thank you!
